### PR TITLE
Add release notes

### PR DESCRIPTION
**Pull request by cangencer**

### CODE REVIEW

**Comment by hz-highfive** [comment]
The job `jet-oss-pr-builder-docs-no-build` of your PR failed ([log](
https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder-docs-no-build/215/build_log.txt), [artifacts](https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder-docs-no-build/215/jet-oss-pr-builder-docs-no-build-215.zip)). 
Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.
<details>
<summary>Click to expand the log file</summary>
<pre>
--------------------------
---------SUMMARY----------
--------------------------
There are unexpected files.
Unexpected file: hazelcast-jet-distribution/src/root/changelog.txt
</details>
</pre>

**Comment by cangencer** [comment]
verify

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 1]
```diff
@@ -1,8 +1,24 @@
 ** Hazelcast Jet 4.2 **
 
-* Generic Debezium-Jet connector and its MySQL & Postgres versions have
-  been wrapped with a proprietary API to make them more convenient to
-  use and more production-ready.
+* We added Change Data Capture (CDC) support for PostgreSQL and MySql. New
+  connectors use Debezium to provide a high-level API for processing a
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:4]: ... processing the ...

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+  stream changes from relational databases. A new map sink can be used to
+  automatically keep the map in-sync with changes from the database.
+* We added new source and sink connectors for Elasticsearch 5, 6 and 7
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:7]: ... and 7.

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
+* We improved JSON support through a new file source which can read JSON files
+  directly and also added built-in Jet methods for parsing and mapping JSON strings.
+* We introduced .relabance() operator which can be used for distributed items
+  across the cluster
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:11]: ... cluster.

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 4]
```diff
@@ -1,8 +1,24 @@
...
+* We made some improvements to Jet startup scripts:
+  - You can now use PROMETHEUS_PORT to start Jet with prometheus metrics enabled
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:13]: ... enabled.

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 5]
```diff
@@ -1,8 +1,24 @@
...
+  - You can use jvm.options file to edit the startup JVM parameters
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:14]: ... use the jvm.options file... ... parameters.

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 6]
```diff
@@ -1,8 +1,24 @@
...
+  - You can now use JET_MODULES environment variable to add modules when using docker
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:15]: ... use the JET_MODULES ...

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 7]
```diff
@@ -1,8 +1,24 @@
...
+    or jet-start script.
+* We did several rounds of improvements to the documentation
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:17]: ... documentation:

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 8]
```diff
@@ -1,8 +1,24 @@
...
+  - Improved Docker and Kubernetes instructions
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:18]: Improvements to Docker ...

File: hazelcast-jet-distribution/src/root/changelog.txt [hunk 9]
```diff
@@ -1,8 +1,24 @@
...
+  - New guidance on Garbage Collector choice and settings
+  - New and improved tutorials for CDC
+  - Added documentation for Python module and early results
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/changelog.txt:21]: New documentation for ...

File: hazelcast-jet-distribution/src/root/release_notes.txt [hunk 10]
```diff
@@ -23,6 +23,47 @@ To expand an issue number into a full GitHub link, use this template:
...
+[elasticsearch] [003] Added source and sink connectors for Elasticsearch 5, 6, 7 (#2098, #2286, #2287)
+
+2. Enhancements
+
+[core] Support Hazelcast Serialization for ProcessorSupplier (#2298)
+[core] Increase default parallelism for file and Avro source to 4 (#2359)
+[pipeline-api] Support for keyFn and valueFn in map sink (#2198)
+[jet-cli] Introduce --targets as the default command for specifying where to connect and
+   add it as a mixins for all comments (@caioguedes #2276)
+[jet-cli] Add support JET_MODULES environment variable to import modules automatically
```
Comment (Serdaro) [inline hazelcast-jet-distribution/src/root/release_notes.txt:50]: Add support for ...

File: hazelcast-jet-distribution/src/root/changelog.txt (hunk 2 above)
Comment (frant-hartm) [inline hazelcast-jet-distribution/src/root/changelog.txt:7]: _... with support for slicing and co-located reading for maximum read performance and extended Elasticsearch's client retry mechanism for reliable writing._

**Pull request merged**